types/json.go

```diff
@@ -230,7 +230,6 @@ func (t *Transaction) UnmarshalJSON(input []byte) error {
 	}
 	*t = Transaction{
 		Version:     uint(jsonObj.Version),
-		Hash:        jsonObj.Hash,
```

Comment:
why delete the `hash` field? I think it's mapping to the rpc type `TransactionView` (  https://github.com/nervosnetwork/ckb/tree/develop/rpc#type-transactionview ) which has the `hash` field. Won't this break the compatibility of the original code? For example:
```
rpc.getBlock("0x...").transactions[0].hash
```

Reply:
The reason I delete `hash` is that I think `hash` is not a necessary field and it can be calculated by the method `Hash()`, and in the marshaling process `hash` should be omitted. After refactoring the code should be like `rpc.getBlock("0x...").transactions[0].Hash()`.